tests/test_validate_edf.py

```diff
@@ -1,12 +1,28 @@
 from pathlib import Path
 
 from electos.ballotmaker import validate_edf
-from electos.ballotmaker.constants import NO_ERRORS, NO_FILE
+from electos.ballotmaker.constants import NO_DATA, NO_ERRORS, NO_FILE
+from genericpath import isfile
 
-test_file = Path("empty.json")
+not_a_file = Path("not_a_file.json")
+test_dir = Path(__file__).parent.resolve()
+# this test JSON files must be in the same dir as this test script
+empty_file = Path("empty.json")
+empty_file_path = Path(test_dir, empty_file)
+test_file = Path("june_test_case.json")
+full_test_path = Path(test_dir, test_file)
```

Comment:
If these are constants, capitalize them,

Reply:
I'm not sure if they're constants, yet. I could see changing the values, especially if I used test fixtures. Also, in my mind, constants are things like strings and ints, not more obscure things like Paths -- not a string. Maybe I'm being old-fashioned. I think this one merits further discussion. I'm going to leave as-is for now.